pex/pip/tool.py

```diff
@@ -925,7 +925,7 @@ def spawn_download_distributions(
         log = None
         popen_kwargs = {}
         if log_analyzers:
-            log = os.path.join(safe_mkdtemp(), "pip.log")
+            log = os.path.join(safe_mkdtemp(prefix="pex-pip-log"), "pip.log")
```

Comment:
This makes this sort of thing easier to debug, you can find the latest of these dirs and then hardlink the pip.log contained within to somewhere non-ephemeral in /tmp to debug the log and not have it disappear. There is plenty of time to do this in these cases since they're pathologically slow!